Extract token to fiat conversion

Gudahtt opened the pull request:
The conversion of token amounts to fiat amounts was extracted from the `TokenCell` component, and moved to a utility function. This will be used elsewhere in an upcoming PR.

Gudahtt committed:
Extract token to fiat conversion

The conversion of token amounts to fiat amounts was extracted from the
`TokenCell` component, and moved to a utility function. This will be
used elsewhere in an upcoming PR.

whymarrh reviewed (changes requested):
Two suggestions in the spirit of [`consistent-return`](https://eslint.org/docs/rules/consistent-return)

Gudahtt committed:
Explicitly return `undefined`

`undefined` is now explicitly returned in the case where no formatted token
fiat amount can be calculated. It was returned implicitly previously.

Co-authored-by: Whymarrh Whitby <whymarrh.whitby@gmail.com>
